config/features.yml

```diff
@@ -505,9 +505,13 @@ features:
   organic_conversion_experiment:
     actor_type: user
     description: Toggle to enable login.gov create account experiment
-  reactivation_experiment:
+  reactivation_experiment_initial_gate:
```

Comment:
The flipper flag clean up task sometimes doesn't work, so you might need to log into the console and run the following to remove the old feature name:
`Flipper.remove("name_of_feature_flag")`

Reply:
Gotcha, I'll double check it once this gets merged and deployed